packages/magento-store/PageMeta.tsx

```diff
@@ -7,10 +7,11 @@ import { StoreConfigDocument } from './StoreConfig.gql'
 
 type PageMetaProps = Pick<NextPageMetaProps, 'title' | 'metaDescription' | 'metaRobots'> & {
   canonical?: string
+  children?: React.ReactNode
```

Comment:
I don't think this isn't necessary, as it is also included in NextpageMetaProps

Reply:
Done